crates/network-scanner/src/scanner.rs

```diff
@@ -262,21 +262,19 @@ impl NetworkScanner {
     }
 }
 
-type ResultReceiver = tokio::sync::mpsc::Receiver<(IpAddr, Option<String>, u16)>;
+pub type ScanEntry = (IpAddr, Option<String>, u16, Option<Protocol>);
+type ResultReceiver = tokio::sync::mpsc::Receiver<ScanEntry>;
 pub struct NetworkScannerStream {
```

Comment:
style: You can consider adding a few newlines to make the code more readable. This is a general comment with this codebase, although it’s not critical.

Reply:
updated